src/components/VideoPlayerModal/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import Modal from 'react-responsive-modal';
+import { VideoPlayer } from '../VideoPlayer';
+
+export interface IVideoPlayerModal {
+  href: string;
+  cta: object;
+}
+
+export const VideoPlayerModal: React.FunctionComponent<IVideoPlayerModal> = ({ href, cta }) => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  const onOpenModal = () => setIsOpen(true);
```

Comment:
This is a good use-case for the useCallback hook! At a high level, the useCallback hook creates an instance of a function once and only re-creates it whenever one of its parameters (the things in the array in the second argument) changes. Without it, every time this component re-renders, it will create a new instance of the function "onOpenModal" which might cause the Modal to re-render unexpectedly. Its good practice to use this hook whenever you're creating a function inside a component when that function should only be re-created on certain (stable) conditions. 🤓 

Check out the docs: https://reactjs.org/docs/hooks-reference.html#usecallback

```suggestion
  const onOpenModal = React.useCallback(() => setIsOpen(true), [setIsOpen]);
```